lib/membrane_rtc_engine/track.ex

```diff
@@ -66,6 +66,7 @@ defmodule Membrane.RTC.Engine.Track do
   @type t :: %__MODULE__{
           type: :audio | :video,
           stream_id: String.t(),
+          payload_type: non_neg_integer() | nil,
```

Comment:
I think that we should introduce RTP mapping field and remove pt and clock rate. We can do this at the end in the last PR but let's leave TODO tag to remember to revisit this place